packages/core/src/routes/experience/classes/libraries/sign-in-experience-validator.ts

```diff
@@ -283,6 +264,11 @@ export class SignInExperienceValidator {
           signUp.identifiers.includes(type) && signUp.verify,
           new RequestError({ code: 'user.sign_up_method_not_enabled', status: 422 })
         );
+
+        assertThat(
```

Comment:
Guard the password requirements here, if the password is enabled, `CodeVerification` can not be used for use registration. A `NewPasswordIdentity` verification is required.